pkg/controller/controller_instance.go

```diff
@@ -40,12 +40,28 @@ func (c *controller) instanceAdd(obj interface{}) {
 		glog.Errorf("Couldn't get key for object %+v: %v", obj, err)
 		return
 	}
-	// TODO(vaikas): If the obj (which really is an Instance right?) has
-	// AsyncOpInProgress flag set, just add it directly to c.pollingQueue
-	// here? Why shouldn't we??
+
 	c.instanceQueue.Add(key)
 }
 
+func (c *controller) instanceUpdate(oldObj, newObj interface{}) {
+	// A polling instance will be added to the polling queue by the reconciler.
+	// We only need to take care of non-polling instances here.
```

Comment:
"non polling" is "not async"?
comment is kind of confusing with mentions of polling in a comment and the code talking about async.